src/main/amp/config/alfresco/module/support-tools/log4j.properties

```diff
@@ -1,273 +1,273 @@
-
-# Dictionary/Model debugging
-log4j.logger.org.alfresco.repo.dictionary=warn
-log4j.logger.org.alfresco.repo.dictionary.types.period=warn
-
-# Virtualization Server Registry
-log4j.logger.org.alfresco.mbeans.VirtServerRegistry=error
-
-# Spring context runtime property setter
-log4j.logger.org.alfresco.util.RuntimeSystemPropertiesSetter=info
-
-# Debugging options for clustering
-log4j.logger.org.alfresco.repo.content.ReplicatingContentStore=error
-log4j.logger.org.alfresco.repo.content.replication=error
-
-#log4j.logger.org.alfresco.repo.deploy.DeploymentServiceImpl=debug
-
-# Activity service
-log4j.logger.org.alfresco.repo.activities=warn
-
-# User usage tracking
-log4j.logger.org.alfresco.repo.usage=info
-
-# Sharepoint
-log4j.logger.org.alfresco.module.vti=info
-
-# Forms Engine
-log4j.logger.org.alfresco.repo.forms=info
-log4j.logger.org.alfresco.web.config.forms=info
-log4j.logger.org.alfresco.web.scripts.forms=info
-
-# CMIS
-log4j.logger.org.alfresco.opencmis=error
-log4j.logger.org.alfresco.opencmis.AlfrescoCmisServiceInterceptor=error
-log4j.logger.org.alfresco.cmis=error
-log4j.logger.org.alfresco.cmis.dictionary=warn
-log4j.logger.org.apache.chemistry.opencmis=info
-
-# IMAP
-log4j.logger.org.alfresco.repo.imap=info
-
-# JBPM
-# Note: non-fatal errors (eg. logged during job execution) should be handled by Alfresco's retrying transaction handler 
-log4j.logger.org.jbpm.graph.def.GraphElement=fatal
-
-#log4j.logger.org.alfresco.repo.googledocs=debug
-
-###### Scripting #######
-
-# Web Framework
-log4j.logger.org.springframework.extensions.webscripts=info
-log4j.logger.org.springframework.extensions.webscripts.ScriptLogger=warn
-log4j.logger.org.springframework.extensions.webscripts.ScriptDebugger=off
-
-# Repository
-log4j.logger.org.alfresco.repo.web.scripts=warn
-log4j.logger.org.alfresco.repo.web.scripts.BaseWebScriptTest=info
-log4j.logger.org.alfresco.repo.web.scripts.AlfrescoRhinoScriptDebugger=off
-log4j.logger.org.alfresco.repo.jscript=error
-log4j.logger.org.alfresco.repo.jscript.ScriptLogger=warn
-log4j.logger.org.alfresco.repo.cmis.rest.CMISTest=info
-
-log4j.logger.org.alfresco.repo.avm.actions=info
-
-# Bulk Filesystem Import Tool
-log4j.logger.org.alfresco.repo.bulkimport=warn
-
-# Freemarker
-# Note the freemarker.runtime logger is used to log non-fatal errors that are handled by Alfresco's retrying transaction handler
-log4j.logger.freemarker.runtime=
-
-# Metadata extraction
-log4j.logger.org.alfresco.repo.content.metadata.AbstractMappingMetadataExtracter=warn
-
-# Reduces PDFont error level due to ALF-7105
-log4j.logger.org.apache.pdfbox.pdmodel.font.PDSimpleFont=fatal
-log4j.logger.org.apache.pdfbox.pdmodel.font.PDFont=fatal
-log4j.logger.org.apache.pdfbox.pdmodel.font.PDCIDFont=fatal
-
-# no index support
-log4j.logger.org.alfresco.repo.search.impl.noindex.NoIndexIndexer=fatal
-log4j.logger.org.alfresco.repo.search.impl.noindex.NoIndexSearchService=fatal
+# Set root logger level to error
```

Comment:
I can't see any change here as the whole file has changed, is this a difference in line endings?